s3torchconnector/src/s3torchconnector/__init__.py

```diff
@@ -1,6 +1,6 @@
 #  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 #  // SPDX-License-Identifier: BSD
-from s3torchconnectorclient import S3Exception
+from s3torchconnectorclient import S3Exception, enable_debug_logging
```

Comment:
As discussed, we'll keep this function inside the `s3torchconnectorclient` library for now.